tests/unit/model.py

```diff
@@ -44,16 +44,19 @@ def configure():
 class TestNic(ModelTest):
 
     def sample_obj(self):
-        return Nic(Node(label='node-99', obm=Ipmi(type="http://schema.massopencloud.org/haas/v0/obm/ipmi",
-                  host= "ipmihost", user= "root", password= "tapeworm")), 'ipmi', '00:11:22:33:44:55')
-        
+        return Nic(Node(label='node-99',
+                        obm=Ipmi(type="http://schema.massopencloud.org/haas/v0/obm/ipmi",  # noqa
```

Comment:
I think add a ``__init__`` in Ipmi and omit ``Ipmi.type`` should be the right one. 

``Ipmi.api_name`` should work and it might be good for now.